src/WebJobs.Script/Config/FunctionsHostingConfigSource.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.Json;
+
+namespace Microsoft.Azure.WebJobs.Script.Config
+{
+    /// <summary>
+    /// Represents a JSON file as an <see cref="IConfigurationSource"/>.
+    /// </summary>
+    public class FunctionsHostingConfigSource : FileConfigurationSource
```

Comment:
naming nit: I like the naming of _Platform_ over _Hosting_ here - as the file you are loading is the "FunctionsPlatformConfig" (by the environment variable).

Reply:
Is the feature in Antares called "hosting config"? /wondering if that is the reason for the name?

Reply:
It's because the config is stored in "hosting" SQL database on stamp level. This name was already discussed previously on initial feature commit.